src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+interface Props {
+  time: string,
+  id?: number
+}
+
+// eslint-disable-next-line
+export class Clock extends React.Component<Props> {
+
+  componentWillUnmount() {
+    clearInterval(this.props.id);
+  }
```

Comment:
you start the clock in one component and you stop it in a different one. not very intuitive